_sass/card.scss

```
@@ -6,6 +6,20 @@
 	border-radius: 5px; /* 5px rounded corners */
 	display: inline-block;
 	margin: 10px;
+
+	& .elastic-fai {
+		font-size: $type-size-1;
+
+		@include breakpoint($small) {
+			font-size: $type-size-3;
+		}
+		@include breakpoint($medium) {
+			font-size: $type-size-4;
+		}
```

```suggestion
		@include breakpoint($medium) {
			font-size: $type-size-2;
		}
```
Just checked in on an IPad, 4 is too small for medium.

---

Great you were able to test. I did not have a proper device here for testing.